api/src/main/java/org/openmrs/Allergy.java

```diff
@@ -183,23 +183,6 @@ public void setSeverity(Concept severity) {
 		this.severity = severity;
 	}
 	
-	/**
-	 * @return Returns the comment
-	 * @deprecated as of 2.3.0, replaced by {@link #getComments()}
-	 */
-	@Deprecated
-	public String getComment() {
-		return getComments();
-	}
-	
-	/**
-	 * @param comment the comment to set
-	 * @deprecated as of 2.3.0, replaced by {@link #setComments(String)}
-	 */
-	@Deprecated
-	public void setComment(String comment) {
-		setComments(comment);
-	}
```

Comment:
Hii @ineshtickoo, kindly fix the travis-ci build failure

As per my observation, you forgot to update the method names in test files.